tasks/src/tasks/store.rs

```diff
@@ -0,0 +1,110 @@
+use std::{collections::HashMap, num::NonZeroU32};
+
+use imap_types::{
+    command::CommandBody,
+    core::Vec1,
+    fetch::MessageDataItem,
+    flag::{Flag, StoreResponse, StoreType},
+    response::{Data, StatusBody, StatusKind},
+    sequence::SequenceSet,
+};
+
+use super::TaskError;
+use crate::Task;
+
+#[derive(Clone, Debug)]
+pub struct StoreTask {
```

Comment:
> The STORE command alters data associated with a message in the
      mailbox.  Normally, STORE will return the updated value of the
      data with an untagged FETCH response.  A suffix of ".SILENT" in
      the data item name prevents the untagged FETCH, and the server
      SHOULD assume that the client has determined the updated value
      itself or does not care about the updated value.

Maybe as doc: `Alter message data.`?